### DISCUSSION TITLE
Txpower on Startup

### DISCUSSION THREAD

**Discussion by koklimabc** [svpcom/wfb-ng#324] [Questions]
As quoted question on "https://github.com/svpcom/wfb-ng/discussions/273",

i did understand the rtl8812au settings is based under "/sys/module/88xxau/parameters/rtw_tx_pwr_idx_override, contents is '20'  but when i check with "iw dev" for "wlan0" , is only shown with 18 dbm only :(, Why?

What is the more effective way to change the tx power during runtime on startup?

**Comment by svpcom** [comment]
Use rtw_tx_pwr_idx_override. It overrides any other settings. Wifi has automatic power control in normal mode, but it is not suitable for wfb-ng links

**Reply by koklimabc** [reply]
pardon, It shld set to other place than "/sys/module/88xxau/parameters/rtw_tx_pwr_idx_override"? Where is it :)?

**Reply by svpcom** [reply]
```
$ cat /etc/modprobe.d/wfb.conf 
# blacklist stock module
blacklist 88XXau
blacklist 8812au
# maximize output power
options 88XXau_wfb rtw_tx_pwr_idx_override=30
```

**Reply by svpcom** [reply]
See Setup HOWTO

**Comment by koklimabc** [comment]
FYI, i'd done everything on '/etc/modprobe.d/wfb.conf' based inside **SSC338Q**, but not luck :(, pls help 
![Screenshot from 2024-02-20 09-06-14](https://github.com/svpcom/wfb-ng/assets/2577622/b23aaa93-a108-4c0c-ba90-b192bd0437de)

**Reply by svpcom** [reply]
Don't see txpower via iw - it will show invalid values if you override it via module parameter

**Reply by koklimabc** [reply]
ok then i will change the txpower at under both "/etc/modprobe.d/wfb.conf" and "/etc/wfb.conf", btw tq :)
